Core/EventHandlers.lua

```diff
@@ -1158,19 +1158,15 @@ function R:OnEvent(event, ...)
 			end
 		end
 
-		-- Handle opening Silver Strongbox & Gilded Chest (Shadowlands, Bastion nodes for Acrobatic Steward toy)
+		-- Handle opening Silver Strongbox & Gilded Chest (Bastiom, Shadowlands nodes for Acrobatic Steward (toy) and Gilded Wader (pet))
```

Comment:
Typo? :D

Reply:
Clearly I have a hard time spelling Bastion. There was not one, but two new variants of this name in this PR alone >_> Fixed in e8415779b087f85212233d4e4e98c9191f229c80